src/main.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App'
-import './index.css'
+import './index.scss'
+import 'macro-css'
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
   <React.StrictMode>
```

Comment:
`StrictMode` is great but it has a bug that re-creates the state twice. Not sure if it is fixed yet, so you can remove it or check if the error is gone